<!-- 
Example launch file showing how to start a Kinect-to-2d-scan node.

You need a Kinect device connected to the system and proper permissions
to open it.  

Comment out the "rviz" line if you don't want the GUI.

You can also run this to see the data:

$ rostopic echo /scan

-->

<launch>
  <node name="reconfigure_gui" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>

  <node 
	name="kinect_fake_2d_scanner" 
	pkg="kinect_2d_scanner"
	type="kinect_2d_scanner_node" 
	output="screen">
    <!-- <param name="port" type="string" value="/dev/ttyACM0"/>  -->
  </node>
  
  <node name="rviz" pkg="rviz" type="rviz" respawn="false" output="screen" 
  /> 
<!-- args="-d $(find kinect_2d_scanner)/launch/kinect_2d_scanner_test.vcg"   -->

</launch>
